http.coffee

```diff
@@ -12,9 +12,12 @@ module.exports.monitor = (logger) ->
 			end.apply(this, arguments)
 			responseTime = process.hrtime(startTime)
 			responseTimeMs = Math.round(responseTime[0] * 1000 + responseTime[1] / 1000)
+			requestSize = parseInt(req.headers["content-length"], 10)
 			if req.route?.path?
 				routePath = req.route.path.toString().replace(/\//g, '_').replace(/\:/g, '').slice(1)
 				Metrics.timing("http_request", responseTimeMs, null, {method:req.method, status_code: res.statusCode, path:routePath})
+				if requestSize
+					Metrics.summary("http_request_size_bytes", requestSize, {method:req.method, path:routePath})
```

Comment:
Maybe add the `status_code` for consistency with the timing metric above?  Could be useful to distinguish between failed and successful requests.